### ISSUE TITLE
Travis CI build

### ISSUE THREAD

**Issue by adamralph** [scriptcs/scriptcs-csharp#7]

**Comment by adamralph** [comment]
I guess this has no value until Roslyn scripting is x-plat and we have some acceptance tests.

**Comment by filipw** [comment]
this is valuable in a sense that we want the project to build successfully outside of windows - having the Travis CI builds will help catch issues with `Path` or other Mono-quirks

**Comment by adamralph** [comment]
Yep, agreed, but I've tried invoking the current roslyn scripting bits (nightly from 4th April) in Mono and it blows up immediately trying to make native calls:

>    [System.DllNotFoundException] clr
>      at (wrapper managed-to-native) Microsoft.CodeAnalysis.GlobalAssemblyCache:GetCachePath (Microsoft.CodeAnalysis.GlobalAssemblyCache/ASM_CACHE,byte*,int&)
>      at Microsoft.CodeAnalysis.GlobalAssemblyCache.GetLocation (ASM_CACHE gacId) [0x00000] in <filename unknown>:0 
>      at Microsoft.CodeAnalysis.GlobalAssemblyCache..cctor () [0x00000] in <filename unknown>:0

If we set up a Travis CI build now it would probably succeed while there are no tests, but as soon as we add some it would fail. I guess there's no harm in putting it in place now. We can always switch it off later if it starts to fail and adds too much noise.

**Comment by filipw** [comment]
https://github.com/dotnet/roslyn/issues/1542

**Comment by adamralph** [comment]
Milestone 1.1 :cry:
